client/scripts/sw-docker-entrypoint

```diff
@@ -113,19 +103,21 @@ welcome() {
     echo "Date: `date -u +%Y-%m-%dT%H:%M:%SZ`"
     echo "Version: `swcli --version`"
     echo "Run: $1 "
-    echo "Model Version: ${SW_SWMP_NAME}@${SW_SWMP_VERSION}"
-    echo "Model Workdir: ${SWMP_DIR}"
-    echo "Runtime Workdir: ${SWRT_DIR}"
+    echo "Model Version: ${SW_MODEL_VERSION}"
+    echo "Runtime Version: ${SW_RUNTIME_VERSION}"
     echo "===================================="
 }
 
 eval_task_prepare(){
     welcome $1
     pre_config
-    pre_check
+    # only remote
+    if [ "${MODE}" == "remote" ]
```

Comment:
use another explicit mode word?  
in the standalone instance, users also can pull the cloud model and runtime.

Reply:
done, use instance url